client-ng1/src/app/js/admin/cfp/cfp.html

```diff
@@ -24,7 +24,30 @@ <h1 translate="view.cfp.home.title"><i class="material-icons cesar-icons-h1">set
                         {{speaker.name ? speaker.name : speaker.firstname + ' ' + speaker.lastname}}
                     </a>
                 </p>
-                <p ng-if="!proposal.valid" class="cesar-error-simple" translate="view.cfp.home.incomplete"></p>
+                <p ng-if="!proposal.valid" class="cesar-error-simple" style="float: left;" translate="view.cfp.home.incomplete"></p>
+                {{  }}
+                <p style="text-align: right;">
+                    <a class="mdl-button mdl-js-button mdl-button--fab mdl-button--mini-fab"
+                       ng-class="{'mdl-button--colored':ctrl.votesMappedByProposalId[proposal.id]==='VETO'}"
+                       ng-click="ctrl.voteVeto(proposal.id); $event.stopPropagation();">
+                        <i class="material-icons mdl-badge">not_interested</i>
```

Comment:
I will change for exposure-neg-1 and exposure-plus-1. But I prefer to keep the other icons to show that it's a different vote that the +1/-1. Are you ok with that ? 
And that's why I did not put -2, -1, +1 and +2 in the DB.